ultranest/popstepsampler.py

```diff
@@ -536,8 +536,219 @@ def __next__(
         else:
             return None, None, None, nc
 
+class PopulationSimpleSliceSampler():
+    """
+       Vectorized Slice sampler without stepping out procedure.
+       In Comparison the PopulationSliceSampler, the sampler calls the likelihood on
```

Comment:
Unlike `:py:class:PopulationSliceSampler`, in `:py:class:PopulationSimpleSliceSampler`, the likelihood is always called with the same number of points.

Reply:
Added to the docstring